pkg/labels/labels.go

```diff
@@ -59,10 +59,29 @@ func (ls Labels) String() string {
 		b.WriteString(strconv.Quote(l.Value))
 	}
 	b.WriteByte('}')
-
 	return b.String()
 }
 
+// Bytes returns ls as a byte slice.
+// It uses an byte invalid character as a separator and so should not be used for printing.
+func (ls Labels) Bytes(buf []byte) []byte {
+	b := bytes.NewBuffer(buf[:0])
+	b.WriteByte('{')
+	for i, l := range ls {
+		if i > 0 {
+			b.WriteByte(',')
+			b.WriteByte(' ')
+		}
+		b.WriteString(l.Name)
+		b.WriteByte('=')
+		b.WriteByte(sep)
+		b.WriteString(l.Value)
+		b.WriteByte(sep)
+	}
+	b.WriteByte('}')
```

Comment:
```suggestion
	b.WriteByte(sep)
```

Reply:
This is for when we chain 2 labels together.

Reply:
Mmh maybe not use SEP as if we do not have existing label it would be confusting